virt/virt-4-12-release-notes.adoc

```diff
@@ -127,6 +124,9 @@ The SVVP Certification applies to:
 
 //CNV-18432 UI create live migration page
 
+//CNV-17809 Live migration with shared storage for work intensive workloads
+* You can now define a dedicated network for live migration to minimize disruption to tenant workloads. To select a network, navigate to *Virtualization* > *Overview* > *Settings* > *Live migration*.
```

Comment:
@stevsmit 
I wanted to point out something small that you missed: ">" should be "->". I noticed it because I make that mistake all the time. I fixed it in one of my PRs, so nothing needs to be done here.